content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -190,6 +190,7 @@ For a reference to old feature gates that are removed, please refer to
 | `ServiceAccountTokenPodNodeInfo` | `false` | Alpha | 1.29 | |
 | `SidecarContainers` | `false` | Alpha | 1.28 | 1.28 |
 | `SidecarContainers` | `true` | Beta | 1.29 | |
+| `SeparateTaintEvictionController` | `true` | Beta | 1.29 | |
```

Comment:
This entry should not appear here. Please make sure the rows are still in alphabetic order.